posthog/session_recordings/queries/test/test_session_recording_list_from_query.py

```diff
@@ -4177,7 +4262,6 @@ def test_top_level_event_host_property_test_account_filter(self):
             first_timestamp=self.an_hour_ago + relativedelta(seconds=30),
             team_id=self.team.id,
             click_count=10,
-            ensure_analytics_event_in_session=False,
```

Comment:
Ah! I don't know why this parameter exists, but this is what was hiding the fact that the not_regex filter didn't match